S/SCIP/build_tarballs.jl

```diff
@@ -0,0 +1,65 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SCIP"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://scip.zib.de/download/release/scipoptsuite-7.0.1.tgz", "971962f2d896b0c8b8fa554c18afd2b5037092685735d9494a05dc16d56ad422")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd scipoptsuite-7.0.1/
+mkdir build
+cd build/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DPAPILO=0 ..
+make
```

Comment:
This will speedup compilation quite a lot
```suggestion
make -j${nproc}
```

Reply:
done in last commit thanks